integration-tests/bats/helper/local-remote.bash

```diff
@@ -136,6 +135,7 @@ SKIP_SERVER_TESTS=$(cat <<-EOM
 ~sql-checkout.bats~
 ~cli-hosted.bats~
 ~profile.bats~
+~log.bats~
```

Comment:
Would really like to have this one on. What doesn't work? How hard is it to fix?

Reply:
Needs checkout to work remotely :(

Reply:
How many of the tests required checkout? It would be worth discussing if we can just skip the tests in remote mode which depend on it.